robot_smach_states/src/robot_smach_states/manipulation/place_designator.py

```diff
@@ -176,14 +176,20 @@ def _create_selection_marker(self, selected_pose):
         marker.lifetime = rospy.Duration(30.0)
         return marker
 
-    def _determine_points_of_interest_with_area(self, entity, area):
+    def _determine_points_of_interest_with_area(self, entity, areades):
         """ Determines the points of interest using an area
         :type entity: Entity
-        :param area: str indicating which volume of the entity to look at
+        :param area: str or str Designator indicating which volume of the entity to look at
         :rtype: [FrameStamped]
         """
 
         # We want to give it a convex hull using the designated area
+        if isinstance(areades, str):
+            area = areades
+        elif areades.resolve_type == str:
+            area = areades.resolve()
+        else:
+            raise AssertionError("Cannot place in {}".format(areades))
```

Comment:
Do we have a support fuction for designators which does this? So if areades is already a string just copy it and if it is a designator resolve it? I can image this would be useful in other situations as well

Reply:
For checking, there is `check_type` in the `designator.checks`-submodule or thereabouts. 
To resolve either a designator or just get the already set concrete value, I don't think there is a util yet for that.